api/agentpb/agent.proto

```diff
@@ -133,6 +133,12 @@ message StartActionRequest {
     string dsn = 1;
     string table = 2;
   }
+  // MongoDBExplainParams describes MongoDB EXPLAIN action parameters.
+  message MongoDBExplainParams {
+    string dsn = 1;
+    string database = 2;
```

Comment:
Can it be a part of DSN?

Reply:
After researching it I found that the `db` part of the dsn is not used as the default db but as the default auth db:
https://github.com/percona/pmm-agent/blob/master/vendor/go.mongodb.org/mongo-driver/x/network/connstring/connstring.go#L253
Example from the command line: 
```
mongo --username=root --password=root-password 127.0.0.1:27017/admin
MongoDB shell version v3.6.8
connecting to: mongodb://127.0.0.1:27017/admin
Implicit session: session { "id" : UUID("77e42c9f-9287-4b90-b9e2-0197b97755da") }
MongoDB server version: 4.0.10-5
```
but if you specify a different db (it already exists in my test)
```
mongo --username=root --password=root-password 127.0.0.1:27017/test
MongoDB shell version v3.6.8
connecting to: mongodb://127.0.0.1:27017/test
Implicit session: session { "id" : UUID("37d249fc-908e-450d-8adc-af70c48c18c0") }
MongoDB server version: 4.0.10-5
WARNING: shell and server versions do not match
2019-09-09T16:33:54.432-0300 E QUERY    [thread1] Error: Authentication failed. :
DB.prototype._authOrThrow@src/mongo/shell/db.js:1608:20
@(auth):6:1
@(auth):1:2
exception: login failed
```
So, we do need the Database parameter